contrib/buildrpm/buildrpmLibfabric.sh

```diff
@@ -132,6 +136,18 @@ usage="Usage: $0 [-i provider_name] [-e provider_name]
   -r    parameter
              add custom RPM build parameter
 
+  -D         install default version file for module
+              {default: do not install version file}
+
+  -P    parameter
```

Comment:
Should probably be `-P prefix` (not `parameter`), and then update the help message to match.  `-c` and `-r` both say `parameter` because they are actually referring to parameters that are passed thru to something else.